packages/js/src/import.js

```diff
@@ -4,17 +4,45 @@ import IndexingService from "./services/IndexingService";
 
 const AioseoV4 = "WPSEO_Import_AIOSEO_V4";
 
-let dropdown, importButton, importForm, spinner, loadingMessage, checkMark;
+let cleanupButton, cleanupDropdown, cleanupForm,
+	importButton, importDropdown, importForm,
+	spinner, loadingMessageCleanup, loadingMessageImport, checkMark;
 
 /**
  * Adds Progress UI elements in the page.
  *
  * @returns {void}
  */
 function addProgressElements() {
-	jQuery( spinner ).insertAfter( importButton );
-	jQuery( checkMark ).insertAfter( spinner );
-	jQuery( loadingMessage ).insertAfter( spinner );
+	jQuery( checkMark ).insertAfter( [ importButton, cleanupButton ] );
+	jQuery( loadingMessageImport ).insertAfter( importButton );
+	jQuery( loadingMessageCleanup ).insertAfter( cleanupButton );
+	jQuery( spinner ).insertAfter( [ importButton, cleanupButton ] );
+}
+
+/**
+ * Function called when importing/cleanup progress is made.
+ *
+ * @param {bool} isImport Whether it's an import.
+ *
+ * @returns {void}
+ */
+function showProgress( isImport ) {
```

Comment:
I'd make this argument a string, so you can call it with `showProgress( 'import' )` or `showProgress( 'cleanup' )` for easier readability.